downloader/src/funTest/kotlin/DirectoryTest.kt

```diff
@@ -52,10 +52,10 @@ class DirectoryTest : StringSpec() {
         "Creates directories for Gradle submodules" {
```

Comment:
Commit message: "Provide values for all values of the identifier" -> "Provide all values for the identifier"?

Reply:
Done.